### PR TITLE
Support customizing the images-dir in images pull/push commands

### PR DESCRIPTION
**Pull request by juamedgod**

### CODE REVIEW

File: cmd/dt/pull.go [hunk 1]
```diff
@@ -103,5 +107,7 @@ func newPullCommand() *cobra.Command {
 		},
 	}
 	cmd.PersistentFlags().StringVar(&outputFile, "output-file", outputFile, "generate a tar.gz with the output of the pull operation")
+	cmd.PersistentFlags().StringVar(&imagesDir, "images-dir", imagesDir,
+		"directory where the images will be pulled to. If not empty, it overrides the default images directory inside the chart directory")
```
Comment (mpermar) [inline cmd/dt/pull.go:111]: Is this description accurate? I thought the images folder was now outside the chart directory after the refactoring recently done.

File: cmd/dt/push.go [hunk 2]
```diff
@@ -79,4 +82,7 @@ func newPushCmd() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.PersistentFlags().StringVar(&imagesDir, "images-dir", imagesDir,
+		"directory containing the images to push. If not empty, it overrides the default images directory inside the chart directory")
```
Comment (mpermar) [inline cmd/dt/push.go:86]: Same comment as above

File: cmd/dt/pull.go (hunk 1 above)
Comment (juamedgod) [reply cmd/dt/pull.go:111]: This only applies to the pull/push commands, where there is no concept of wrap bundle, so the images are still placed in the chart dir. The refactoring was for the wrap/unwrap, where we control the format of the bundle.

File: cmd/dt/push.go (hunk 2 above)
Comment (juamedgod) [reply cmd/dt/push.go:86]: Same response :)

File: cmd/dt/pull.go (hunk 1 above)
Comment (mpermar) [reply cmd/dt/pull.go:111]: Ah, right I recall it now. I guess it makes sense. I wonder if the images-dir should be mandatory. Anyways, users will tell. Looks good then.

**Review by mpermar — APPROVED** [review]

**Pull request merged**